core-bundle/src/Image/Studio/Figure.php

```diff
@@ -60,7 +60,6 @@ class Figure
      * All arguments but the main image result can also be set via a Closure
      * that only returns the value on demand.
      *
-     * @param ImageResult                                                                 $image          Main image
```

Comment:
Why this? :upside_down_face: 

Isn't the description helpful? Or if not, why keep it with the other parameters?

Reply:
I don't know exactly why the CS fixer has removed it (probably because there is a type hint already), but no, I don't consider the description "Main image" very helpful. Especially as there is no secondary image or something alike that could make people misinterpret the parameter called `$image`. No hard feelings. 🙈

Reply:
Well there is a secondary image (lightbox) but it is derived. Fine for me anyways. :relaxed: